docs/pages/docs/query/client.mdx

```diff
@@ -0,0 +1,110 @@
+---
+title: "Client"
+description: "Query a Ponder app with the `@ponder/client` package."
+---
+
+import { Callout, Steps, Tabs } from "nextra/components";
+
+# `@ponder/client`
+
+The `@ponder/client` package is a TypeScript client for querying a Ponder app with end-to-end type safety.
+
+## Example projects
+
+These example apps demonstrate how to use `@ponder/client`.
+
+- [**Basic**](https://github.com/ponder-sh/ponder/blob/kjs/client-db/examples/with-client/client/src/index.ts)
+- [**NextJs**](https://github.com/ponder-sh/ponder/blob/kjs/client-db/examples/with-nextjs/frontend/src/hooks/useDeposits.ts)
+
+## Get started
+
+<Steps>
+
+### Update to `>=0.9.0`
+
+`@ponder/client` is available starting from version `0.9.0`. Read the [migration guide](/docs/migration-guide#090) for more details.
+
+### Register client middleware
+
+```ts filename="src/api/index.ts" {7}
+import { db } from "ponder:api";
+import { Hono } from "hono";
+import { client } from "ponder";
```

Comment:
I wonder if this and `graphql` should come from a `ponder/middleware` entrypoint